apps/docs/app/layout.config.tsx

```
@@ -15,29 +18,66 @@
 import { AiOutlineDiscord as DiscordIcon } from 'react-icons/ai'
 import { SiStorybook as StorybookIcon } from 'react-icons/si'
 
-/**
- * Shared layout configurations
- *
- * you can configure layouts individually from:
- * Home Layout: app/(home)/layout.tsx
- * Docs Layout: app/docs/layout.tsx
- */
+const ICONS = new Map<string, JSX.Element>([
+  ['Website', <LandingPageIcon />],
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)